enctests/testframework/utils/utils.py

```diff
@@ -168,6 +168,9 @@ def get_source_metadata_dict(source_clip):
 
 def create_clip(config):
     path = Path(config.get('path'))
+    if not path.is_absolute():
+        if 'config_path' in config:
+            path = Path(os.path.dirname(config.get("config_path")), path)
```

Comment:
I'm  afraid this might be a bit too rigid. Relative paths may also include "../../some/path/" which will fail in this case.
Perhaps if we could do something like:
```
config_path = Path(config.get('config_path'))
path = config_path.parent.joinpath(path).resolve()
```